src/components/location/location.edgedb.repository.ts

```diff
@@ -0,0 +1,70 @@
+import { Injectable } from '@nestjs/common';
+import {
+  ID,
+  NotImplementedException,
+  PaginatedListType,
+  PublicOf,
+  UnsecuredDto,
+} from '~/common';
+import { e, RepoFor } from '~/core/edgedb';
+import { CreateLocation, Location, LocationListInput } from './dto';
+import { LocationRepository } from './location.repository';
+
+@Injectable()
+export class LocationEdgeDBRepository
```

Comment:
@CarsonF - I think this has to do with me including `mapImage` in the `hydrate` method, but it needs to be there.  How can I handle this?

Reply:
```diff
-       mapImage: false, //TODO - add this to hydrate
+       mapImage: loc.mapImage.id,
```

Reply:
Ah wait that doesn't work.
We'll need to change away from `DefinedFile` first.

Reply:
Kinda glad it wasn't that easy. lol.  Yeah I was playing around with `DefinedFile` but didn't get anywhere.